### PR TITLE
Partial update for the Sky Palace update

### PR DESCRIPTION
**Pull request by w0en**
Changes made: 
- Added MP/Es for the 14 new SP mice
- Added "Peggy the Plunderer" to the pirateSealMice array
- Added "Empyrean Javelineer" to the dragons array

Yet to be implemented:
- Base AR for Extra Rich Cloud Cheesecake (the new bait)
- Mouse Populations for SP and the new phases that come with it

### CODE REVIEW

**Comment by w0en** [comment]
Changed Power Type Effectiveness for all power types effective on Richard the Rich from 100 to 200, in accordance with the August 17 Patchnotes (https://www.mousehuntgame.com/newspost.php?news_post_id=570).

File: src/main/cre.js [hunk 1]
```diff
@@ -839,7 +841,7 @@ function icebergPhase() {
     autoPhase = "Bombing Run";
   else if (
     phaseName === "Bombing Run (Ultimate Iceberg)" &&
-    baseName !== "Ultimate Iceberg Base"
+    (baseName !== "Ultimate Iceberg Base" || baseName !== "Iceberg Boiler Base")
```
Comment (DevJackSmith) [inline src/main/cre.js:844]: This logic is always true @w0en

File: src/main/cre.js [hunk 2]
```diff
@@ -859,7 +861,7 @@ function icebergPhase() {
     autoPhase = "Treacherous Tunnels";
   else if (
     phaseName === "Treacherous Tunnels (Ultimate Iceberg)" &&
-    baseName !== "Ultimate Iceberg"
+    (baseName !== "Ultimate Iceberg Base" || baseName !== "Iceberg Boiler Base")
```
Comment (DevJackSmith) [inline src/main/cre.js:864]: Same here @w0en

File: src/main/cre.js [hunk 3]
```diff
@@ -886,7 +888,7 @@ function icebergPhase() {
     autoPhase = "The Mad Depths";
   else if (
     phaseName === "The Mad Depths (Ultimate Iceberg)" &&
-    baseName !== "Ultimate Iceberg"
+    (baseName !== "Ultimate Iceberg Base" || baseName !== "Iceberg Boiler Base")
```
Comment (DevJackSmith) [inline src/main/cre.js:891]: And here

**Comment by DevJackSmith** [comment]
There also seem to be conflicts which need to be resolved before this or can be merged. Let me know if you want help resolving them @w0en

File: src/main/cre.js (hunk 1 above)
Comment (tsitu) [reply src/main/cre.js:844]: That's my bad 😅

File: data/mouse-power-effs.csv [hunk 4]
```diff
@@ -1067,4 +1067,18 @@ Paragon of the Lawless,124100,0,0,0,0,0,0,0,0,500,0
 Paragon of Strength,305875,0,0,0,0,0,500,0,0,0,0
 Paragon of Shadow,246650,0,0,0,0,0,0,500,0,0,0
 Paragon of Tactics,246965,0,0,0,0,0,0,0,500,0,0
-Richard the Rich,33725,100,100,100,100,0,100,100,100,100,0
+Richard the Rich,33725,200,200,200,200,0,200,200,200,200,0
+Glamorous Gladiator,28575,0,0,0,0,0,600,0,0,0,0
+Zealous Academic,28000,0,0,0,0,0,0,600,0,0,0
+Rocketeer,28575,0,0,0,0,0,0,0,600,0,0
```
Comment (tsitu) [inline data/mouse-power-effs.csv:1073]: Rocketeer and Sky Glider MPs don't match what's shown in this sheet: https://docs.google.com/spreadsheets/d/14dw0_YOnIanEaGkm3ABllDxytghJyyiXOp3FX0be1xw/edit#gid=521526777

Comment (tsitu) [reply data/mouse-power-effs.csv:1073]: @w0en

Comment (w0en) [reply data/mouse-power-effs.csv:1073]: Changed and pushed

**Review by DevJackSmith — APPROVED** [review]

**Comment by DevJackSmith** [comment]
merging this in a day or two if no objections/changes

**Pull request merged**